.env.dev.sample

```diff
@@ -1,3 +1,3 @@
-; To use dev env ini file.
-; cp .env.dev.sample .env
+# To use dev env ini file.
```

Comment:
I have replaced with `#` since the the semicolon was showing warning messages when running the application